tranzito_utils_js/dist/dropdown_period_selector.js

```diff
@@ -0,0 +1,53 @@
+import flatpickr from 'flatpickr/dist/flatpickr'
+import { urlForPeriod } from '../utils/period_select'
+
+class DropdownPeriodSelector {
+  init () {
+    this.initDropdownPeriodSelector()
+  }
+
+  initDropdownPeriodSelector () {
+    document.getElementById('periodSelectDropdown')
+      .addEventListener('click', () => {
+        const selectedPeriod = document.querySelector('#periodSelectDropdown .dropdown-item.active a')
+          .attributes['data-period'].value
+        if (selectedPeriod === 'custom') this.initCustomPeriodSelector()
+      })
+    this.enablePeriodSelection()
+  }
+
+  enablePeriodSelection () {
+    document.querySelectorAll('#periodSelectDropdown .dropdown-item').forEach((dropdwonLink) => {
+      dropdwonLink.addEventListener('click', (e) => {
+        if (dropdwonLink.children[0].attributes['data-period'].value === 'custom') e.preventDefault()
```

Comment:
I think you meant `dropdownLink`, but you have consistently spelled it `dropdwonLink`, so the code will work, and maybe there's a reason 😄 ?

Reply:
Ah.. my bad. I misspelled it once and copy pasted it 😄 Sorry for that.